lib/src/filter/change_color.dart

```diff
@@ -0,0 +1,21 @@
+import '../image.dart';
+import '../internal/clamp.dart';
+
+/// Change the color of the [src] image.
+///
+/// [red] defines the red color of the image, as an integer (0 - 255).
+///
+/// [green] defines the green color of the image, as an integer (0 - 255).
+///
+/// [blue] defines the blue color of the image, as an integer (0 - 255).
+Image changeColor(Image src, {int red, int green, int blue}) {
+  var pixels = src.getBytes();
+
+  for (var i = 0, len = pixels.length; i < len; i += 4) {
+    pixels[i] = clamp255((red).toInt());
```

Comment:
* What happens if red, green, or blue is null?  Does null.toInt() work?

* Is it intended that RGB is always set, so it's just filling the image (minus the alpha channel) with a solid color, or are the red, green and blue values optional? If it always sets the RGB channels, then is it basically the same as 'fill' but leaves the alpha channel alone? If that's the case, maybe it'd be better to update the fill function to make the alpha channel optional. If it's optionally changing the red, green, and/or blue channels, then it seems like there should be null checks here.

* The clamping should be done outside of the for loop.